dlite_entities_service/cli/_utils/global_settings.py

```diff
@@ -63,10 +71,37 @@ def global_options(
             "Use the .env file at the given location for the current command. "
             "By default it will point to the .env file in the current directory."
         ),
+        show_default=True,
+        rich_help_panel="Global options",
+    ),
+    token: OptionalStr = typer.Option(
+        None,
+        "--token",
+        help="The token to use for authentication.",
         show_default=False,
         rich_help_panel="Global options",
     ),
 ) -> None:
     """Global options for the CLI."""
-    if dotenv_path is not None:
+    if dotenv_path:
         CONTEXT["dotenv_path"] = dotenv_path
+
+    if token:
+        access_token = Token(access_token=token)
+
```

Comment:
This is a draft PR but this should be removed in the final version

Reply:
Sorry this part is fine, I meant the long sect of commented out code below:

        # I cannot come up with a scenario where this would not validate.
        # The only scenario in which it would happen is if `token` is not a string.
        # But it can never not be a string due to the way the input is parsed.
        # Even the `if token:` sentence ensures that it will never be an empty string at
        # that...
        # However, should a case ever come up, the following lines should be the
        # 'except' part of a 'try/except'-block for generating the `Token()`:

        # except ValidationError as exc:
        #     raise typer.BadParameter(
        #         f"Invalid token: {token}",
        #         param=token,
        #         param_hint="Token should be given as a string.",
        #     ) from exc

Reply:
ahh.. hehehe. Yeah, that's fair. I would need to not squash & merge then, to keep this code alive in the git history though then. But that's fine, I suppose :) 👍

Reply:
Note: I have removed this now, except the actual comment, which I've moved up immediately prior to the line `access_token = Token(...)` and changed accordingly.

I think the comment is useful still. But everything from `# However, should a case ever come up ...` has been removed.